demos/gallery/samples/html/FieldsetForm.js

```diff
@@ -0,0 +1,71 @@
+import { Bar } from "@hpcc-js/chart";
+
+new Bar()
+    .target("target")
+    .columns(["Category","Value"])
+    .data([
+        ["A", 3],
+        ["B", 5],
+        ["C", 8]
+    ])
+    .render()
+    ;
```

Comment:
Doesn't look like a suitable sample?

Reply:
Travis was throwing errors when trying to use FieldsetForm in this sample... it think it was related to Travis trying to pull sources from unpkg? I'll have to revisit this.